VotingApplication/VotingApplication.Web/Content/Scss/Voting.scss

```diff
@@ -118,14 +118,34 @@ body {
 
```

Comment:
I have pulled out CSS for each of the directives to make them stand-alone (i.e. we can put them anywhere in the site). Perhaps it would be worth considering doing so for this?